lib/hooks/use-member-info.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Member, useEcomApi } from '../ecom';
+
+export function useMemberInfo() {
```

Comment:
already mentioned this above, I'd rather use `user` instead of `member` in the code base.